test/parallel/test-http-timeout.js

```diff
@@ -23,13 +23,15 @@
 require('../common');
 
 const http = require('http');
+const Countdown = require('../common/countdown');
 
 const server = http.createServer(function(req, res) {
   res.writeHead(200, { 'Content-Type': 'text/plain' });
   res.end('OK');
 });
 
 const agent = new http.Agent({ maxSockets: 1 });
+const countdown = new Countdown(11, () => server.close());
```

Comment:
Could you store 11 in a `const`? Thanks!

Reply:
@apapirovski : Its not being referenced anywhere else in the file. Do you still want me to store it in a `const` ?

Reply:
@apapirovski : Update the PR to address the review comment and also shortened the commit message. Kindly review now.Thanks!